snarkos/main.rs

```diff
@@ -119,7 +119,7 @@ async fn start_server(config: Config) -> anyhow::Result<()> {
     info!("Storage finished loading");
 
     // Enable the sync layer.
-    {
+    if !node.config.is_bootnode() {
```

Comment:
Bootnodes need the mempool enabled for the time being, this will break broadcast transaction on Aleo Explorer otherwise

Reply:
```suggestion
{
```

Reply:
Reverted 👍